public/utils/constants.ts

```diff
@@ -38,3 +38,15 @@ export const APP_PATH = {
 export const PLUGIN_NAME = 'opendistro-anomaly-detection';
 
 export const ALERTING_PLUGIN_NAME = 'opendistro-alerting';
+
+export const ANOMALY_RESULT_INDEX = '.opendistro-anomaly-results';
+
+export const MAX_DETECTORS = 1000;
+
+export const MAX_ANOMALIES = 10000;
+
+export const DETECTOR_STATE = Object.freeze({
+  DISABLED: 'Disabled',
+  INIT: 'Initializing',
+  RUNNING: 'Running',
+});
```

Comment:
We should be able to just use a TypeScript enum here. From what I can tell Object.freeze() is a JavaScript workaround to create similar behavior to enums. See https://github.com/ohltyler/anomaly-detection-kibana-plugin/blob/updateDetectorListPage/public/pages/utils/constants.ts#L18

Reply:
sure. will use the same type as yours.

Reply:
+1